projects/tests.py

```diff
@@ -394,11 +390,16 @@ def test_invalid_project(self):
         self.assertEqual(resp.status_code, 404)
 
     def test_invalid_project_permissions(self):
+        user, _ = User.objects.get_or_create(username='invalid_perms')
+        token, _ = Token.objects.get_or_create(user=user)
+
+        self.client.force_authenticate(user=user, token=token)
```

Comment:
This is how you can either authenticate as a different user for a particular test. If you want to remove authentication, you can set `user` and `token` to `None` to ensure the request is submitted without any credentials.